Runtime/UIElements/Controls/LoadingSpinner.cs

```diff
@@ -0,0 +1,90 @@
+﻿using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace StansAssets.Foundation.UIElements
+{
+    /// <summary>
+    /// The LoadingSpinner control. let's you place buttons group with the labels or images
```

Comment:
let's you place buttons group with the labels or images?

Reply:
no, it's just a spinner. you can embed it in any control you want